src/pages/Overview/TrafficManagement.tsx

```diff
@@ -0,0 +1,288 @@
+import * as React from 'react';
+import { Button, Modal } from '@patternfly/react-core';
+import NamespaceInfo from './NamespaceInfo';
+import { AuthorizationPolicy, Sidecar } from 'types/IstioObjects';
+import { MessageType } from 'types/MessageCenter';
+import { PromisesRegistry } from 'utils/CancelablePromises';
+import { DurationInSeconds } from 'types/Common';
+import { IstioConfigPreview } from 'components/IstioConfigPreview/IstioConfigPreview';
+import * as AlertUtils from 'utils/AlertUtils';
+import * as API from 'services/Api';
+import { serverConfig } from '../../config';
+import GraphDataSource from 'services/GraphDataSource';
+import {
+  buildGraphAuthorizationPolicy,
+  buildNamespaceInjectionPatch,
+  buildGraphSidecars
+} from 'components/IstioWizards/WizardActions';
+import { AUTHORIZATION_POLICIES } from '../IstioConfigNew/AuthorizationPolicyForm';
+
+type TrafficManagementProps = {
+  opTarget: string;
+  kind: string;
+  isOpen: boolean;
+  nsTarget: string;
+  nsInfo: NamespaceInfo;
+  hideConfirmModal: () => void;
+  load: () => void;
+  duration: DurationInSeconds;
+};
+
+type State = {
+  confirmationModal: boolean;
+  authorizationPolicies: AuthorizationPolicy[];
+  sidecars: Sidecar[];
+  disableOp: boolean;
+  canaryVersion: string;
+};
+
+export default class TrafficManagement extends React.Component<TrafficManagementProps, State> {
```

Comment:
I read "TrafficManagement" a little bit confusing in this context.
I suggest "TrafficPolicies" or "OverviewTrafficPolicies" as in this folder specific components for this page are used in that way.

In future iterations it can be refactored in other way.